src/Parser/AfflictionWarlock/Modules/SoulShards/SoulShardDetails.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import SoulShardBreakdown from './SoulShardBreakdown';
+import SoulShardTracker from './SoulShardTracker';
+
+import WastedShardsIcon from '../../Images/affliction_warlock_soulshard_bw.jpg';
+
+const soulShardIcon = 'inv_misc_gem_amethyst_02';
+
+class SoulShardDetails extends Module {
+  static dependencies = {
+    soulShardTracker: SoulShardTracker,
+  };
+
+  suggestions(when) {
+    const shardsWasted = this.soulShardTracker.shardsWasted;
+    when(shardsWasted).isGreaterThan(0)
```

Comment:
Suggestions are generally based around amount per minute/second/percentage of fight instead of static amounts. The suggestion occurrence will differ based on fight duration which may not be ideal.